spec/index.html

```diff
@@ -1802,42 +1814,59 @@ <h3>RDF Triples Constructors</h3>
+      <a href="#grammar-production-reifier"><code>reifier</code></a> of the
+      <a href="#handle-reifiedTriple"><code>reifiedTriple</code></a> term constructor
+      if present,
+      or a fresh RDF blank node, if not matched.
       A new <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> instance |TT|
       is composed from
       the <a href="#grammar-production-subject"><code>subject</code></a>,
       <a href="#grammar-production-predicate"><code>predicate</code></a>, and
       <a href="#grammar-production-object"><code>object</code></a> productions.
-      Finishing the <a href="#grammar-production-reifier"><code>reifier</code></a> production
+      Finishing the <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       yields the RDF triple |curSubject| <code>rdf:reifies</code> |TT|
       and restores the recorded values of |curSubject| and |curPredicate|.</p>
 
-    <h4 id="annotation" style="padding-bottom:0; margin-bottom:0;"><span>Annotations:</span></h4>
+    <h4 id="annotation" style="padding-bottom:0; margin-bottom:0;"><span>Annotations</span></h4>
 
     <p>Beginning the <a href="#grammar-production-annotation"><code>annotation</code></a> production
       records the |curSubject| and |curPredicate|.
-      A new <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> instance |TT|
-      is created using |curSubject| |curPredicate| |curObject|.
-      The identifier |TI| is taken from the
-      <a href="#handle-annotation"><code>annotation</code></a> term constructor.
+      A new <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> instance |curTripleTerm|
+      is created using |curSubject| |curPredicate| |curObject|,
+      and the value of |curReifier| is reset.
       Finishing the <a href="#grammar-production-annotation"><code>annotation</code></a> production
-      yields the RDF triple |TI| <code>rdf:reifies</code> |TT|
+      yields the RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|,
+      resets the value of |curReifier|,
       and restores the recorded values of |curSubject| and |curPredicate|.</p>
-    <p class="ednote">Describe parser state for recording and assigning any <a>reifier</a>.</p>
 
-    <h4 id="propertyList" style="padding-bottom:0; margin-bottom:0;"><span>Property Lists:</span></h4>
+    <h4 id="annotationBlock" style="padding-bottom:0; margin-bottom:0;"><span>Annotation Block</span></h4>
+
+    <p>Beginning the <a href="#grammar-production-annotationBlock"><code>annotationBlock</code></a> production,
+      |curReifier| is taken from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>, if not already set,
+      and |curSubject| is taken from |curReifier|
+      and yields the RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|,
+      and resets the value of |curReifier|.</p>
```

Comment:
```suggestion
      if |curReifier| is not set, then |curReifier| is assigned a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a> and then yield RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|.
      and |curSubject| is taken from |curReifier|
      and yields the RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|,
      and clears the value of |curReifier|.</p>
```
i.e. only yield if curReifier is not set. Previous setting by an explicit reifier already yielded the `rdf:reifies` triple.
This copes with 
```
:s :p :o :~r .
:s :p :o :~r {| ... |} 
:s :p :o {| ... |} :~r .
:s :p :o {| ... |} 
:s :p :o {| 
       :p :q ~:r2 {|  ... |} ;
       :p :q ~:r23{|  ... |} ;
   |} .
:s :p :o ~:r {| ... |}  {| ... |}
```
The last case - the annotates production has a star so clearing `curReifier`on exit does not clear it per loop but it still needs clearing because a reifier may be last.